### DISCUSSION TITLE
Should summary view specify frequency band for dB level?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/bioacoustic-dashboard#9] [Polls]
I have a pet peeve about reporting or visualizing sound pressure levels without specifying _both_ the reference pressure and the frequency band (and for source levels the reference distance.) The current SeaStats `Summary view` does the former, but not the latter:

<img width="432" alt="Screenshot 2023-12-06 at 1 25 03 PM" src="https://github.com/orcasound/bioacoustic-dashboard/assets/14044595/6a4dc33d-2df1-4d38-89d5-d7705417ca8f">

**Comment by scottveirs** [comment]
Any strong opinions, @veirs or @ben-hendricks ?

**Reply by ben-hendricks** [reply]
The "personas" using SeaStats will be different and some users already mentioned the lack of information on these types of details.
I think it should be added, but set back from the main information to not crowd the screen. Maybe below dB re 1muPa in a grey shade and smaller font size: [100 - 1000 Hz]?

**Comment by ben-hendricks** [comment]
On another note: I believe the content of the "landing page" should be re-worked entirely in 2024. We had reasons for a simple design in 2023 but I think we can do better with more time to plan.
